extension/src/time_vector.rs

```diff
@@ -30,8 +33,10 @@ pub mod toolkit_experimental {
         struct Timevector<'input> {
             num_points: u32,
             is_sorted: bool,
-            internal_padding: [u8; 3],  // required to be aligned
+            flags: u8,         // extra information about the stored data
+            internal_padding: [u8; 2],  // required to be aligned
             points: [TSPoint; self.num_points],
+            null_val: [u8; (self.num_points + 7)/ 8], // bit vector, must be last element for alignment purposes
```

Comment:
Ah, if only [`div_ceil()`](https://doc.rust-lang.org/stable/std/primitive.u64.html#method.div_ceill) was stable, then we could use that instead of doing this manually. I do wonder if we should extract this out into a type, or at least extract out `(num_points + 7)/ 8` into a function